app/code/core/Maho/Blog/Helper/Data.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+/**
+ * Maho
+ *
+ * @category   Maho
+ * @package    Maho_Blog
+ * @copyright  Copyright (c) 2025 Maho (https://mahocommerce.com)
+ * @license    https://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+class Maho_Blog_Helper_Data extends Mage_Core_Helper_Abstract {}
```

Comment:
Reviewing on my phone so don't have exact code suggestion, but you can define a protected module name property equal to 'Maho_Blog' for translations to use Maho_Blog.csv